internal/kv/mem/mem_kv.go

```diff
@@ -44,7 +45,7 @@ func (kv *MemoryKV) Load(key string) (string, error) {
 	item := kv.tree.Get(memoryKVItem{key, ""})
 	// TODO，load unexisted key behavior is weird
 	if item == nil {
-		return "", nil
+		return "", fmt.Errorf("key %s is not exist", key)
```

Comment:
reject for now. As the config items used in helm are not the same as YAML files located in the configs directory.

Reply:
Yep.